server/database/models/User.js

```diff
@@ -2,28 +2,27 @@ const { Schema, model } = require("mongoose");
 const bcrypt = require("bcryptjs");
 const jwt = require("jsonwebtoken");
 
-var userSchema = new Schema({
-  name: {
-    type: String,
-    required: true
-  },
-  email: {
-    type: String,
-    required: true
-  },
-  password: {
-    type: String,
-    required: true
-  },
-  experience: {
-    type: Schema.Types.Mixed,
-    of: Number
+var userSchema = new Schema(
+  {
```

Comment:
Something changed with the formatting here, make sure we have all the same prettier.io settings on back-end too